docs/changelog/134221.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 134221
```

Comment:
Let's label this as a non-issue? Because released versions will not run with doc value skippers enabled And when the feature flag gets removed then we label it as a feature or enhancement.